cpr/session.cpp

```diff
@@ -291,6 +293,25 @@ void Session::Impl::SetVerifySsl(const VerifySsl& verify) {
     }
 }
 
+void Session::Impl::SetProtocolVersion(const ProtocolVersion& protocolversion) {
+#if LIBCURL_VERSION_MAJOR >= 7
+#if LIBCURL_VERSION_MINOR >= 33
+#if LIBCURL_VERSION_PATCH >= 0
+    auto curl = curl_->handle;
+    if (curl) {
+        if (protocolversion == cpr::HTTP::v2) {
```

Comment:
currently only setting the protocol version if it's http/2; is it important to set http/1.0 or http/1.1? what if you want to reuse a `Session` across both http/1 and http/2 requests?

Reply:
Agreed here, let's let users set any version, not just go from `default` to `v2`